link/src/test/java/com/stripe/android/link/ui/signup/SignUpScreenshotTest.kt

```diff
@@ -0,0 +1,140 @@
+                emailController = EmailConfig.createController("test@test.com"),
+                phoneNumberController = PhoneNumberController.createPhoneNumberController("5555555555"),
+                nameController = NameConfig.createController("Jane Doe"),
+                signUpScreenState = SignUpScreenState(
+                    signUpEnabled = false,
+                    signUpState = SignUpState.InputtingPrimaryField,
+                    merchantName = "Example Inc.",
+                    requiresNameCollection = true
+                ),
+            ) {
+            }
+        }
+    }
+
+    @Test
+    fun status_inputting_email_shows_only_email_field3() {
+        snapshot {
+            SignUpBody(
+                emailController = EmailConfig.createController("test@test.com"),
+                phoneNumberController = PhoneNumberController.createPhoneNumberController("5555555555"),
+                nameController = NameConfig.createController("Jane Doe"),
+                signUpScreenState = SignUpScreenState(
+                    signUpEnabled = false,
+                    signUpState = SignUpState.InputtingPrimaryField,
+                    merchantName = "Example Inc.",
+                    requiresNameCollection = false
+                ),
+            ) {
+            }
+        }
+    }
+
+    @Test
+    fun status_inputting_email_shows_only_email_field4() {
+        snapshot {
+            SignUpBody(
+                emailController = EmailConfig.createController("test@test.com"),
+                phoneNumberController = PhoneNumberController.createPhoneNumberController("5555555555"),
+                nameController = NameConfig.createController("Jane Doe"),
+                signUpScreenState = SignUpScreenState(
+                    signUpEnabled = false,
+                    signUpState = SignUpState.InputtingPrimaryField,
+                    merchantName = "Example Inc.",
+                    requiresNameCollection = true,
+                    errorMessage = "Something went wrong".resolvableString
+                ),
+            ) {
+            }
+        }
+    }
+
+    @Test
+    fun status_inputting_email_shows_only_email_field5() {
+        snapshot {
+            SignUpBody(
+                emailController = EmailConfig.createController("test@test.com"),
+                phoneNumberController = PhoneNumberController.createPhoneNumberController("5555555555"),
+                nameController = NameConfig.createController("Jane Doe"),
+                signUpScreenState = SignUpScreenState(
+                    signUpEnabled = false,
+                    signUpState = SignUpState.VerifyingEmail,
+                    merchantName = "Example Inc.",
+                    requiresNameCollection = true,
+                    errorMessage = "Something went wrong".resolvableString
+                ),
+            ) {
+            }
+        }
+    }
+
+    @Test
+    fun status_inputting_email_shows_only_email_field6() {
```

Comment:
can you please name these tests something more descriptive than ending with a number?